packages/atomic/src/pages/examples/ipx.html

```diff
@@ -225,19 +279,24 @@
           <atomic-layout-section section="pagination">
             <atomic-load-more-results class="pagination-more-result"></atomic-load-more-results>
           </atomic-layout-section>
+          <atomic-recs-interface fields-to-include='["cat_rating_count"]'>
```

Comment:
Is there a reason to why `cat_rating_count` should be included?

Reply:
Not really! I just let it here as an example for our setup in the Search Interface Service. I'll remove the field and let the array empty.

Reply:
Removed ✅!